kythe/extractors/bazel/extract.sh

```diff
@@ -14,103 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Script to extract compilations from a Bazel repository.  See usage() below.
+bazel "$@"
```

Comment:
This pretty radically changes the requirements for running this script.

Whereas before it would work outside of a docker image, it now *requires* the exact filesystem layout of the Docker image and that the requisite bazel build has already been run.  This is a big step backwards.

Reply:
Sure maybe we can change /workspace/output to be $KYTHE_OUTPUT_DIRECTORY mimicing what I have in gcp land, which works locally.

For /kythe/kzip, I have no immediate great ideas.  I've been stuffing things into /opt/kythe where things used to install (still do, too? /hedberg), and referencing those from either inside or outside of docker.  But I have no preference for /opt/kythe vs /kythe

Could require kzip tool to live on $PATH, as discussed offline maybe.  (Or have a $KYTHE_TOOL_DIRECTORY)

Reply:
I suspect we should have a design document around things like this, KYTHE_PATH falling back to PATH if that's missing for binary locations seems reasonable. The same with KYTHE_OUTPUT_DIRECTORY, although for a script like this having it be a command line flag with a suitable usage message is more user friendly.